configuration/rules-dsl.md

```diff
@@ -430,6 +430,13 @@ calendar.timeInMillis = now.millis
 val dtt = new DateTimeType(calendar)
 ```
 
+Note: as DateTypeType(calendar) above is now deprecated, the simplest non-deprecated way to update a DateTimeType item with a Joda DateTime value in a rule is by converting it to a string.
```

Comment:
DateTimeType?

Reply:
Oh, true - it should just read "... a DateTime item ...". Now this n00b has to figure out how to edit the proposed edit.

Reply:
And also the DateTypeType. Clearly I was not paying attention.